lesson4/index.html

```diff
@@ -0,0 +1,194 @@
+				</li>
+				<li class="header__item">
+					<a class="header__link" href="#">Projects</a>
+				</li>
+				<li class="header__item">
+					<a class="header__link" href="#">Contact</a>
+				</li>
+			</ul>
+		</div>
+	</header>
+	<div class="preview">
+		<div class="preview__wrapper centre">
+			<img class="preview__page1" src="img/page3.png">
+			<img class="preview__page2" src="img/page2.png">
+			<img class="preview__page3" src="img/page1.png">
+			<div class="preview__info">
+				<div class="preview__title">
+					Lorem ipsum dolor sit
+				</div>
+				<div class="preview__text">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+					tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+					quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+					consequat. 
+				</div>
+				<button class="preview__button">See More</button>
+			</div>
+		</div>
+	</div>
+	<div class="info">
+		<div class="subscription centre">
+			<div class="subscription__text">
+				Lorem ipsum dolor sit amet, consectetur adipisicing elit. Porro, consequatur, rem.
+			</div>
+			<form class="subscription__form">
+				<input class="subscription__email" type="email" name="email" placeholder="Email">
+				<input class="subscription__submit" type="submit" name="dd" value="Submit">
+			</form>
+		</div>
+		<div class="articles centre">
+			<div class="container">
+				<div class="container__title">
+					<img class="container__picture" src="img/container1.png">
+					<span>Ultricies Lacinea Pulvinar</span>
+				</div>
+				<div class="container__text">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda quae, illum, facilis ab provident dolores debitis corrupti quod nihil itaque quasi eligendi error omnis!
+				</div>
+				<button class="container__button">
+					See more
+				</button>
+			</div>
+			<div class="container">
+				<div class="container__title">
+					<img class="container__picture" src="img/container2.png">
+					<span>Magnust Minima sed quia</span>
+				</div>
+				<div class="container__text">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda quae, illum, facilis ab provident dolores debitis corrupti quod nihil itaque quasi eligendi error omnis!
+				</div>
+				<button class="container__button">
+					See more
+				</button>
+			</div>
+			<div class="container">
+				<div class="container__title">
+					<img class="container__picture" src="img/container3.png">
+					<span>Ultricies lacinia pulvinar</span>
+				</div>
+				<div class="container__text">
+					Lorem ipsum dolor sit amet, consectetur adipisicing elit. Assumenda quae, illum, facilis ab provident dolores debitis corrupti quod nihil itaque quasi eligendi error omnis!
+				</div>
+				<button class="container__button">
+					See more
+				</button>
+			</div>
+			<div class="container">
+				<img class="container__picture--big" src="img/container4.png">
```

Comment:
container__picture container__picture--big - it is important to use both, but I didn't find any styles for bid. Do you need it?